server/opendp_project/urls_cypress.py

```diff
@@ -0,0 +1,12 @@
+"""
+!DANGER!: These urlpatterns should only be used in test.
+Please read server/opendp_project/settings/cypress_settings for an explanation
+"""
+from django.urls import path, include
+
+from opendp_project.urls import urlpatterns
+from opendp_apps.cypress_utils.check_setup import are_cypress_settings_in_place
+
+if are_cypress_settings_in_place:
```

Comment:
Is there a way for us to test spinning up an instance that shouldn't have this endpoint, then verifying that trying to hit it returns a 404, and similarly that the management command doesn't exist?

Reply:
There are several tests here that receive a 404 when hitting the endpoint and also test the managment command:
  - https://github.com/opendp/dpcreator/blob/9e3faa05059fbf144d47b8068eddabaef189d192/server/opendp_apps/cypress_utils/tests.py

Reply:
Thanks for pointing that out, looks great